config/locales/doc_auth/en.yml

```
@@ -22,6 +22,7 @@ en:
           taking a new picture.
         birth_date_checks: We couldn’t read the birth date on your ID. Try taking
           new pictures.
+        birth_date_min_age: Your birth date does not meet the minimum age requirement.
```

Is this final copy? I never use the term "birth date", what about something like:

```suggestion
        birth_date_min_age: Your birthday does not meet the minimum age requirement.
```